core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java

```diff
@@ -161,6 +157,27 @@
  *
  * @author Gerrit van Brakel
  * @since 4.2
+ *
+ */
+/*
+ * The receiver is the trigger and central communicator for the framework.
+ * <br/>
+ * The main responsibilities are:
+ * <ul>
+ *    <li>receiving messages</li>
+ *    <li>for asynchronous receivers (which have a separate sender):<br/>
+ *            <ul><li>initializing ISender objects</li>
+ *                <li>stopping ISender objects</li>
+ *                <li>sending the message with the ISender object</li>
+ *            </ul>
```

Comment:
Hier kan je volstaan met 'sending the response by means of the sender'

Reply:
Dit is gewoon commentaar dat ik voor jullie bewaard heb. Frank ontwikkelaars zien het niet meer.